styles/bauhaus.css

```
@@ -99,12 +101,7 @@ th,
   margin-top: 15px;
 }
 
-/*=========================== LAYOUT =========================*/
-
-#invoice {
-  border-top: 5px solid #000;
-  padding: 0 1cm 1cm 1cm;
-}
+/*=========================== LAYOUT =========================*/ß

```

Trailing German character here that's been overlooked.
```suggestion
/*=========================== LAYOUT =========================*/

```